### PR TITLE
FDN-643-Set-up-PL-and-readPreference-map

### PR DESCRIPTION
**Pull request by mbarcia**

### CODE REVIEW

File: api/api.js [hunk 1]
```diff
@@ -93,7 +87,7 @@ module.exports = async (req, res) => {
     res.setHeader('Set-Cookie', `${process.env.TITLE}=null;HttpOnly;Max-Age=0;Path=${process.env.DIR || '/'}`)
 
     // Remove logout parameter.
-    res.setHeader('location', req.url && decodeURIComponent(req.url).replace(/logout\=true/, ''))
+    res.setHeader('location', req.url && decodeURIComponent(req.url).replace(/logout=true/, ''))
```
Comment (mbarcia) [inline api/api.js:90]: Fix suggested by IntelliJ, thought it was spot on

File: api/api.js [hunk 2]
```diff
@@ -118,7 +112,7 @@ module.exports = async (req, res) => {
   req.params.user = user
 
   // Retrieve path component from request URL for method routing.
-  const path = req.url.match(/(?<=\/api\/)(.*?)[\/\?]/)
+  const path = req.url.match(/(?<=\/api\/)(.*?)[\/?]/)
```
Comment (mbarcia) [inline api/api.js:115]: Fix suggested by IntelliJ, thought it was spot on

File: api/api.js [hunk 3]
```diff
@@ -25,12 +25,6 @@ const routes = {
   provider: provider,
 }
 
-process.env.COOKIE_TTL = process.env.COOKIE_TTL || 3600
```
Comment (mbarcia) [inline api/api.js:28]: Removed these defaults, as they now live in the Dockerfile

File: mod/dbs.js [hunk 4]
```diff
@@ -14,9 +17,17 @@ module.exports = () => {
         return aurora(key.split('_')[1], process.env[key]);
     });
 
-  function postgres(key, connectionString) {
+  async function postgres(key, connectionString) {
```
Comment (mbarcia) [inline mod/dbs.js:20]: I had to make the function async

**Review by gabrielncorreia — APPROVED** [review]

**Pull request merged**